lib/danger/commands/init.rb

```diff
@@ -244,7 +244,7 @@ def unsure_token
     end
 
     def note_about_clicking_links
-      ui.say "Note: Holding cmd ( ⌘ ) and #{ENV['ITERM_SESSION_ID'] ? '' : 'double '}clicking a link will open it in your browser."
```

Comment:
That's what I recon too since it's in the Ruby docs. @segiddins probably knows for sure?